app/src/main/java/openfoodfacts/github/scrachx/openfood/models/entities/store/StoreResponse.kt

```diff
@@ -0,0 +1,26 @@
+package openfoodfacts.github.scrachx.openfood.models.entities.store
+
+import openfoodfacts.github.scrachx.openfood.models.entities.EntityResponse
+
+
+class StoreResponse (
+    private var code: String,
+    private var names: Map<String, String>,
+    private val wikiDataCode: String? = null
+) : EntityResponse<Store> {
+    override fun map(): Store {
+        val store: Store
+        if (wikiDataCode != null) {
+            store = Store(code, arrayListOf(), wikiDataCode)
+            names.forEach { (key, value) ->
```

Comment:
Can we use .apply{} here? And then inline the return.

Reply:
We can use .also{}